src/components/interfaces/MOBILE_API.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="SmartDeviceLink RAPI" version="5.1.0" minVersion="1.0" date="2019-03-19">
+<interface name="SmartDeviceLink RAPI" version="5.2.0" minVersion="1.0" date="2019-5-22">
```

Comment:
Seems that changing the date is a bit unnecessary, better just to update it after the spec is finalized.